databases/week2/part1.sql

```diff
@@ -0,0 +1,70 @@
+--Part 1
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+   task(title, description, created, updated, due_date, status_id, id)
+ VALUES
+   ("HYF courses", "meet Sunday", "2022-03-17 21:27:00", "2022-03-19 11:11:17", "2022-03-18 10:01:16", 2, 36);
+
+--Change the title of a task
+UPDATE task
+SET title ="HYF database week2"
+WHERE id = 36;
+
+--Change a task due date
+UPDATE task
+SET due_date = "2022-03-20 12:00:00"
+WHERE id =36;
+
+--Change a task status
+UPDATE task
+SET status_id = 1
+WHERE id = 36;
+
+--Mark a task as complete
+
+UPDATE task
+SET status_id = 3
+WHERE id = 36;
+
+--Delete a task
+DELETE FROM task
+WHERE id = 35;
+
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.*
```

Comment:
formatting suggestion
SELECT [COLUMNS]
FROM [TABLE NAME]
   JOIN [TABLE NAME] + [JOIN CONDITION]
WHERE [CONDITION]

Reply:
see other occurrences and update